stories/exercises/06_exercise.mdx

```diff
@@ -0,0 +1,53 @@
+# 06. Customize components using settings.
+
+In the last exercise, we familiarized ourselves with the USWDS theme settings. In this section, we'll take that a step further by using component specific settings.
+
+Each USWDS component with settings will have this section. It is a snapshot of what customizations are possible without needing to write custom styles.
+
+**Tip:** The [Component settings](https://designsystem.digital.gov/documentation/settings/#component-settings-2) section of the Settings page provides a comprehensive list of all available component settings, their default values, and what type of token or value they take!
+
+1. Visit the [banner component](https://designsystem.digital.gov/components/banner/) page. Find the "Banner settings" section.
+
+1. To get started, lets try updating the font to a serif font. In `uswds-theme.scss`, add the following line to your settings map:
+    ```scss
+    $theme-banner-font-family: "serif",
+    ```
+1. Easy, right? Let's add a bit more color to our banner. Let's see what would happen if we brought back our accent color:
+    ```scss
+    $theme-banner-background-color: "accent-cool"
+    ```
+    The banner background color updated, but if you take a close at the banner link the contrast doesn't look great. If you take a look at the script running in the terminal window you'll notice a bunch of color contrast warnings have popped up.
```

Comment:
Consider adding the message incase terminal isn't up.

```
   ```sh
       Warning: [Banner] Neither the specified preferred link color token (`base-lighter`) nor the fallback link color token (`primary`) have AA contrast on a `accent-cool` background. Using `primary`. Please check your source code and project settings.
       node_modules/@uswds/uswds/packages/uswds-core/src/styles/functions/color/get-color-token-from-bg.scss 65:7   get-color-token-from-bg()
    ```
```

Reply:
Resolved in [0b26483](https://github.com/Bixal/storybook-rapid-prototyping/pull/19/commits/0b26483ce4bd6fcdb6ba701474c29e69315df43c)